mlir/test/Conversion/EmulateFp8ExtTrunc/Integration/roundtrip.mlir

```diff
@@ -53,7 +69,20 @@ func.func @testAllExtTrunc() {
     %o = arith.bitcast %oF8 : ##TYPE## to i8
     %mismatch = arith.cmpi ne, %i, %o : i8
     scf.if %mismatch {
-      func.call @printFail(%i, %o, %iFloat) : (i8, i8, f32) -> ()
+      // 7d, 7e, and 7f are all NaN.  They become 7f800001 as f32
```

Comment:
That isn't correct for the old types - their NaN is `0x80`

Reply:
It only opens a window for a missed failure when input is 7d or 7e and output is 7f.  But I put a conditional in anyway.